services/Storage/storageAccounts/alerts.yaml

```diff
@@ -380,21 +383,21 @@
 - name: fileServices - FileCapacity
```

Comment:
This one needs the reference link @maheshbenke :

https://learn.microsoft.com/en-us/azure/storage/files/files-monitoring-alerts#how-to-create-an-alert-if-the-azure-file-share-size-is-80-of-capacity